### PR TITLE
Add CGI toggle to user interface

### PR DESCRIPTION
**Pull request by iliajie**

### CODE REVIEW

File: edit_website.cgi [hunk 1]
```diff
@@ -88,6 +88,16 @@ if (!$d->{'alias'} && &can_log_paths() &&
 			&ui_textbox("elog", $elog, 60));
 		}
 	}
+# CGI execution mode
+my @cgimodes = &has_cgi_support();
+if (@cgimodes > 0) {
```
Comment (jcameron) [inline edit_website.cgi:93]: Maybe change it to `> 1` since there's no point offering this option if the user only has one choice

Comment (iliajie) [reply edit_website.cgi:93]: What about Disable mode?

Comment (jcameron) [reply edit_website.cgi:93]: What I mean is, if the system doesn't support fcgiwrap or suexec, then the only option will be "disabled" which it doesn't make much sense to show.

Comment (iliajie) [reply edit_website.cgi:93]: Yeah, but `&has_cgi_support();` doesn't return disabled state ..

**Comment by jcameron** [comment]
Thanks for the patch!

**Pull request merged**